src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Ensuring Security Amidst the xz Backdoor Concern
+description:
+date: 2024-04-02
+cover: /images/blog/enhancing-type-safety.png
+timeToRead: 7
+author: jake-barnby
+category: engineering
+featured: false
+---
+
+In the light of recent unsettling revelations regarding a backdoor discovered in the widely-used xz utility, a compression tool used in Linux environments, including Red Hat and Debian systems, the cyber-security landscape has been abuzz with concern. This discovery had a large potential impact on encrypted SSH connections, a backbone of secure communications in the tech world.
+
+At Appwrite, ensuring the security and trust of our developers and users is paramount. We understand the concerns that arise from such vulnerabilities and their potential implications. It's crucial for our community to know that Appwrite's services remain unaffected by the xz backdoor. This issue affected beta and test versions of Red Hat and Debian distributions, which Appwrite does not use.
+
+# What Does This Mean for Self-Hosted Appwrite Solutions?
```

Comment:
We need to update all title cases.

Reply:
Title should be h2

Reply:
The markdown pre-processor turns all headings 1 level lower, this is rendered as H2